src/MultiFormGoals/ProgressBar/Model.php

```diff
@@ -132,14 +133,18 @@ public function getDonationRevenueResults()
     /**
      * Get raw earnings value for Progress Bar
      *
+     * @unreleased use DonationQuery
      * @since 2.9.0
      */
     public function getTotal(): string
     {
-        $query = new Query($this->getForms());
-        $results = $query->getResults();
+        $amount = 0;
 
-        return Money::ofMinor($results->total, give_get_option('currency'))->getAmount();
+        foreach($this->getForms() as $formId) {
```

Comment:
@alaca what do you think about updating the query to accept multiple form IDs? We could update the `where` to be a `whereIn`.

Reply:
That's a great idea. I added a new method `forms` where you can pass an array of form IDs.  See https://github.com/impress-org/givewp/pull/7380/commits/35eb5346024fafd85e9ee7653346fb08b698ba1e